### PR TITLE
French translations for plugin manager

### PR DESCRIPTION
**Pull request by Mysterious-Dev**
All is in the title ^^.

This changes can be skipped for the changelog ^^.

### Desired reviewers

 @dduportal or @smerle33 or @lemeurherve or @gounthar or @olamy

### CODE REVIEW

**Review by lemeurherve — APPROVED** [review]
Thanks!

File: core/src/main/resources/hudson/util/HudsonIsRestarting/index_fr.properties
Comment (dduportal) [inline core/src/main/resources/hudson/util/HudsonIsRestarting/index_fr.properties:23]: Good catch 👏

**Review by dduportal — APPROVED** [review]
Thanks! LGTM.

Please note that this repository also need validation from legit maintainers (which we are not, except for @olamy maybe) and follow the usual Jenkins Core PRs review process.

**Comment by gounthar** [comment]
Failed after more than 1 hour of build on the JDK21 build stage because of  `Caused by: java.lang.OutOfMemoryError: Java heap space`.

**Comment by MarkEWaite** [comment]
> Failed after more than 1 hour of build (total) on the JDK21 build stage because of `Caused by: java.lang.OutOfMemoryError: Java heap space`.

That's been an infrequent failure mode for the Java 21 build environments.  I've been using retry as the temporary solution.  I think we may need to open an infra ticket to help isolate the problem and resolve it.

**Comment by Mysterious-Dev** [comment]
@NotMyFault All tests passed ^^. So I think it can be marked as ready for merge like #8684 ^^

**Review by NotMyFault — APPROVED** [review]
/label ready-for-merge

---

This PR is now ready for merge. We will merge it after ~24 hours if there is no negative feedback.
Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process.
Thanks!

**Pull request merged**
